### PR TITLE
FIX #1983 : Updated Css to move disabled button smoothly

### LINKED ISSUE #1983: [Bug] Moving a disabled button needs to be simpler
You can't move a disabled button, you need to either enable it, or resize it [or get lucky =) ]. 

Please refer attached gif

Bug_1

### Important Details
 - Version: Cloud 
 - OS: MacOSX
 - Browser : Firefox 82.0.2 (64-bit)

### PR DESCRIPTION
**Pull request by vibhandikyash**
## Description
Moving a disabled button was difficult. Replaced css. Tested in firefox.

Fixes #1983 

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

> Tested locally.
> Tested in Firefox and Chrome (Mac).

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

## Test coverage results :test_tube:
<details><summary>:white_circle: Total coverage has not changed</summary>

    // Code coverage diff between base branch:release and head branch: FIX/1983-moving-disabled-button-made-simpler 
Status | File | % Stmts | % Branch | % Funcs | % Lines 
 -----|-----|---------|----------|---------|------ 
 :green_circle: | app/client/src/components/designSystems/blueprint/ButtonComponent.tsx | 59.74 **(0.53)** | 52.63 **(-0.1)** | 66.67 **(0)** | 61.97 **(0.54)**</details>

### CODE REVIEW

**Review by somangshu — DISMISSED** [review]

**Comment by somangshu** [comment]
@vivekverma2312 please test this out properly.

**Comment by vivekverma2312** [comment]
@vibhandikyash Currently I am not able to drag and drop any widget on another widget. If we drag and drop container widget on canvas then I am not allowed to drag and drop any other widget on top of it. Similarly the button on Forms are also not clickable and can not be moved.

https://www.loom.com/share/41098cb5b3604a56a005175beba8c75a

File: app/client/src/components/editorComponents/DraggableComponent.tsx [hunk 1]
```diff
@@ -171,6 +170,8 @@ function DraggableComponent(props: DraggableComponentProps) {
         transform: `translate(-50%, -50%)`,
         top: "50%",
         left: "50%",
+        pointerEvents:
```
Comment (marks0351) [inline app/client/src/components/editorComponents/DraggableComponent.tsx:173]: We should be handling this scenario in the button widget itself. use the disabled state to disable button actions(click or submit) without setting the CSS styles like pointer-events and button attributes like [disabled].

**Review by marks0351 — CHANGES_REQUESTED** [review]

**Review by marks0351 — DISMISSED** [review]

**Comment by somangshu** [comment]
Flaky test fail, Re-ran the jobs

**Comment by somangshu** [comment]
@vibhandikyash merge latest release here

**Review by somangshu — APPROVED** [review]

**Comment by somangshu** [comment]
Tests still failing

**Comment by somangshu** [comment]
@vibhandikyash any update here?

**Review by somangshu — APPROVED** [review]
Approving to run test, Need to check if this is the right way to change the test

**Comment by somangshu** [comment]
@vibhandikyash latest release merge required here!

**Comment by somangshu** [comment]
@vibhandikyash latest merge required here for fix failing tests.

**Review by somangshu — APPROVED** [review]

**Comment by vivekverma2312** [comment]
@vibhandikyash Deploy preview is tested and it's working fine.

**Pull request merged**